pkg/client/results/testdata/mockResults/plugins/job-junit-02/job-junit-02.golden.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"name":"job-junit-02","status":"failed","items":[{"name":"output.xml","status":"passed","meta":{"file":"results/global/output.xml"},"items":[{"name":"testsuite-001","status":"passed","items":[{"name":"[k8s.io] Pods should be submitted and removed [NodeConformance] [Conformance]","status":"passed"},{"name":"[sig-node] ConfigMap should fail to create ConfigMap with empty key [Conformance]","status":"passed"},{"name":"[sig-storage] Downward API volume should set DefaultMode on files [LinuxOnly] [NodeConformance] [Conformance]","status":"passed"},{"name":"[sig-storage] In-tree Volumes [Driver: local][LocalVolumeType: dir-link-bindmounted] [Testpattern: Dynamic PV (default fs)] subPath should support existing directories when readOnly specified in the volumeSource","status":"skipped"},{"name":"[sig-storage] In-tree Volumes [Driver: rbd][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] subPath should support restarting containers using file as subpath [Slow]","status":"skipped"}]}]},{"name":"output2.xml","status":"failed","meta":{"file":"results/global/output2.xml"},"items":[{"name":"testsuite-001","status":"failed","items":[{"name":"[k8s.io] Pods should be submitted and removed [NodeConformance] [Conformance]","status":"passed"},{"name":"[sig-apps] Daemon set [Serial] should rollback without unnecessary restarts [Conformance]","status":"failed"},{"name":"[sig-storage] In-tree Volumes [Driver: local][LocalVolumeType: dir-link-bindmounted] [Testpattern: Dynamic PV (default fs)] subPath should support existing directories when readOnly specified in the volumeSource","status":"skipped"},{"name":"[sig-storage] In-tree Volumes [Driver: rbd][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] subPath should support restarting containers using file as subpath [Slow]","status":"skipped"}]}]}]}
```

Comment:
The main thing that changes in all of these is that the entire 'testsuite' is put into a new level. This way when you have N test suites, each is on their own branch and can be filtered/walked appropriately.

Reply:
There are a number of these golden files where it looks like the failure messages aren't being recorded. They seem to get captured correctly in `junit_good_junit.xml` file though.

Reply:
💯 thanks for the catch. When developing I do tend to be a little less than critical with my review of those goldenfiles.

Probably worth it to do something like change (for tests) to MarshalIndent (with 0 indent so changes in depth don't cascade to every sub item) so that the diffs are more clean.